src/webelement.rs

```diff
@@ -368,12 +368,109 @@ impl<'a> WebElement<'a> {
         convert_json(&v["value"])
     }
 
+    /// Return true if the WebElement is currently displayed, otherwise false.
+    ///
+    /// # Example
+    /// ```rust
+    /// # use thirtyfour::prelude::*;
+    /// # use thirtyfour::support::block_on;
+    /// #
+    /// # fn main() -> WebDriverResult<()> {
+    /// #     block_on(async {
+    /// #         let caps = DesiredCapabilities::chrome();
+    /// #         let driver = WebDriver::new("http://localhost:4444/wd/hub", &caps).await?;
+    /// #         driver.get("http://webappdemo").await?;
+    /// #         let elem = driver.find_element(By::Id("button1")).await?;
+    /// let visible = elem.is_visible().await?;
+    /// #         assert_eq!(visible, true);
+    /// #         Ok(())
+    /// #     })
+    /// # }
+    /// ```
+    pub async fn is_visible(&self) -> WebDriverResult<bool> {
```

Comment:
Why not use the W3C name is_displayed()?

Reply:
ah, yeah I did that originally and changed it to be consistent with the issue. I prefer `is_displayed` because it's more consistent with the spec. I'll change it back